packages/server/src/api/controllers/market-data.ts

```diff
@@ -30,7 +41,24 @@ function getPoolWeeklyOHLC(req: Request<unknown, unknown, unknown, GetMarketData
     return BitqueryFetcher.getLastWeekOHLC(baseToken, quoteToken);
 }
 
+// GET /marketData/daily
+async function getPoolIndicators(req: Request<unknown, unknown, unknown, GetIndicatorsQuery>) {
+    const { baseToken, quoteToken, days } = req.query;
+
+    const numDays = days ? parseInt(days, 10) : 19;
```

Comment:
you can set `.default(19)` in the Joi schema. also I don't think you need to parse because Joi is doing that already. You can remove this line if you set the default.

Reply:
✅